src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -1396,12 +1317,12 @@ internal void ResolveConflict([@NotNull]string docID, [@CanBeNull]IConflictResol
                     {
                         // Do this in a batch so that there are no changes to the document between
                         // localDoc read and remoteDoc read
-                        localDoc = new Document(this, docID);
+                        localDoc = new Document(_defaultCollection, docID);
```

Comment:
What happens if `_defaultCollection` is null?

Reply:
This is a good question. We were allowing null database before, I wonder if we are also allowing null collection as the parameter? If not, how should we handle null default collection case? Just return false in readSuccess?

Reply:
Or treat doc isn't existed?

Reply:
I changed to use DefaultCollection which will fetch the default collection from LiteCore.